webview/src/views/settings.svelte

```diff
@@ -14,13 +16,14 @@
   /**
    * Event listener for saving the settings
    */
-  async function saveSettings(e: Event) {
+  async function saveSettings() {
     // TODO: handle errors
-    if (!(e.target instanceof HTMLFormElement)) {
+    const form = document.getElementById("settings-form")
```

Comment:
Why use IDs when you can make this dynamic?

Reply:
Because this was being triggered by an `on:click` event on a button the `event.target` was not the form and we needed to get it explicitly. This code has been removed in [1c79ea5](https://github.com/temporalio/vscode-debugger-plugin/pull/18/commits/1c79ea5ac64702549a57c0338d9feab48a521e50).

Reply:
👍